langchain/document_loaders/googledrive.py

```diff
@@ -171,13 +195,16 @@ def _load_document_from_id(self, id: str) -> Document:
         }
         return Document(page_content=text, metadata=metadata)
 
-    def _load_documents_from_folder(self, folder_id: str) -> List[Document]:
+    def _load_documents_from_folder(self, folder_id: str,
+                                    file_types: Optional[str] = None) -> List[Document]:
         """Load documents from a folder."""
         from googleapiclient.discovery import build
 
         creds = self._load_credentials()
         service = build("drive", "v3", credentials=creds)
         files = self._fetch_files_recursive(service, folder_id)
+        if file_types:
+            files = [file for file in files if file["mimeType"] in file_types]
```

Comment:
is `mimeType`  a mime type (application/pdf) or a simplified file type (e.g,. `pdf`) ?

Reply:
`mimeType` is a full mime type (application/pdf). If the user gave simplified file types (eg `pdf`), they were translated to full mime types in `validate_inputs`